code/game/turfs/open.dm

```diff
@@ -63,7 +64,16 @@
 				return 0
 			if(C.m_intent=="walk" && (lube&NO_SLIP_WHEN_WALKING))
 				return 0
-
+		if(istype(C, /mob/living/carbon/human))
```

Comment:
im pretty sure only humans have shoes. 

the next if could be reduced, though.